js/count-chars.js

```diff
@@ -12,24 +12,32 @@
 
 var aiosp_title_extra = parseInt( aioseop_count_chars.aiosp_title_extra );
 
-jQuery( document ).ready( function() {
-	aioseopInitCounting();
-});
+jQuery( document ).ready(
```

Comment:
This doesn't need to be broken down. Everyone generally expects `jQuery( document ).ready( function() {`

Reply:
I'm not sure about what everyone expects but I believe this is correct as it is an anonymous function defined within a function and that needs indentation. It is not a breaking change and if it improves readability, why not? 

Also, the only way to exclude such things that deviate from the definition is to add comments on the lines that need to be excluded which I think will add more clutter. I don't think this change is a big deal.

Reply:
It does not improve readability, if anything it throws it off. Everyone expects `jQuery( document ).ready( function() {`, and it's just redefine a coding style that no one uses...but us...